salt/states/docker_network.py

```diff
@@ -110,7 +129,11 @@ def present(name, driver=None, containers=None):
     else:
         try:
             ret['changes']['created'] = __salt__['docker.create_network'](
-                name, driver=driver)
+                name,
+                driver=driver,
+                driver_opts=driver_opts,
+                check_duplicate=True)
```

Comment:
IIUC, this way ``check_duplicate`` will always be ``True`` whenever ``docker_network.present`` is called.
Seems logical to me as we don't want this state to produce a duplicated network.

Thinking further, be there some failure (race?) in salt to [whether the network already exist](https://github.com/saltstack/salt/blob/develop/salt/states/docker_network.py#L96),  this duplication may occur. So LGTM having it set to ``True`` :-)

Thank you.